src/ReaderScreen/utils/gutkaScript.js

```diff
@@ -101,20 +120,33 @@ window.addEventListener(
   false
 );
 
-${listener}.onload = () => {
+// Use window.onload for Android reliability
+window.onload = () => {
   if (${theme.mode === "dark"}) {
   //fade event
 fadeInEffect();
 }
 
-  scrollToPosition(); 
+  scrollToPosition();
+  
+  // Minimal scrollbar - works on Android and iOS
+  const existingScrollbar = document.getElementById("sb");
+  if (existingScrollbar) existingScrollbar.remove();
+  scrollbar = document.createElement("div");
+  scrollbar.id = "sb";
+  const scrollbarThumb = document.createElement("div");
+  scrollbarThumb.id = "sb-t";
+  scrollbar.appendChild(scrollbarThumb);
+  document.body.appendChild(scrollbar);
+  setTimeout(updateScrollbar, 100);
+  window.addEventListener("resize", updateScrollbar);
```

Comment:
Everytime the `window.onload` fires, it adds a new resize listener without removing the previous one. Maybe store its reference somewhere, and use that to add a condition that if the listener already exists, remove it before adding a new one.